packages/extension-base/src/koni/background/handlers/Extension.ts

```diff
@@ -1593,6 +1593,26 @@ export default class KoniExtension {
     return historySubject.getValue().filter((item) => addresses.some((address) => isSameAddress(item.address, address)));
   }
 
+  private async subscribeHistoryByChainAndAddress ({ address, chain }: RequestSubscribeHistory, id: string, port: chrome.runtime.Port): Promise<ResponseSubscribeHistory> {
+    const cb = createSubscription<'pri(transaction.history.subscribe)'>(id, port);
+
+    const historySubject = await this.#koniState.historyService.getHistorySubject();
```

Comment:
Please response data in this subscriptions only:
const subscription = await this.#koniState.historyService.subscribeHistories(chain, address, cb);